src/pages/academy/adminPanel/subcomponents/CourseConfigPanel.tsx

```diff
@@ -57,12 +67,33 @@ const CourseConfigPanel: React.FC<CourseConfigPanelProps> = props => {
     />
   );
 
+  const writePanel1 = (
+    <TextArea
+      id="defaultPrompt"
+      className="input-textarea"
+      fill={true}
+      value={defaultPrompt || ''}
+      onChange={e =>
+        props.setCourseConfiguration({
+          ...props.courseConfiguration,
+          defaultPrompt: e.target.value
+        })
+      }
+    />
+  );
+
   const previewPanel = (
     <div className="input-markdown">
       <Markdown content={moduleHelpText || ''} openLinksInNewWindow />
     </div>
   );
 
+  const previewPanel1 = (
+    <div className="input-markdown">
+      <Markdown content={defaultPrompt || ''} openLinksInNewWindow />
+    </div>
+  );
+
```

Comment:
There's a lot of code duplication here. We should not duplicate code unnecessarily. Please refactor to reduce the code duplication.